compiler-plugin/src/main/kotlin/arrow/meta/quotes/filebase/File.kt

```diff
@@ -44,7 +61,7 @@ class File(
   val virtualFilePath: String = value.virtualFilePath,
   val danglingAnnotations: ScopedList<KtAnnotationEntry> = ScopedList(value = value.danglingAnnotations, postfix = ", "),
   val fileType: FileType = value.fileType,
-  val declaractions: ScopedList<KtDeclaration> = ScopedList(value = value.declarations, postfix = ", "),
+  val declarations: ScopedList<KtDeclaration> = ScopedList(value = value.declarations, postfix = ", "),
   val stub: KotlinFileStub? = value.stub,
```

Comment:
This one also requires more work, it's definitely one of the more versatile elements that requires more examples